pkg/apis/core/v1alpha1/storagecluster.go

```diff
@@ -128,6 +129,19 @@ type SelfSignedSpec struct {
 	SharedSecret  *string        `json:"sharedSecret,omitempty"`
 }
 
+// GuestAccessType lets the user choose the
+// level of permissions the system.user has
+type GuestAccessType string
+
+var (
```

Comment:
I feel we should keep this const so there are no accidental changes to the value of the enums at runtime.
We can always a pointer to the copy of this constant. That's what is done in kubernetes for types like HostPathType or MountPropagationMode in volumes.

Reply:
Fixed